lib/service/service.go

```diff
@@ -1694,6 +1701,27 @@ func (process *TeleportProcess) initAuthService() error {
 	}
 	authServer.SetLockWatcher(lockWatcher)
 
+	if authServer.Embeddings != nil {
+		log.Infof("Starting embedding watcher")
```

Comment:
```suggestion
		log.Debugf("Starting embedding watcher")
```
Let's try not to log in `Info` without a reason.